src/utils/Logger.h

```diff
@@ -120,7 +104,7 @@ class Logger {
 
 	// don't allow copy-construction
 	Logger(const Logger&) : _log_level(Log::Error), _msg_log_level(Log::Error), _do_output(true),
-			_filename(""), _log_stream(nullptr), logLevelNames(), _starttime(), _rank(0)
+			_log_stream(nullptr), logLevelNames(), _starttime(), _rank(0)
 	{ }
```

Comment:
If we don't want to allow this shouldn't we then just declare this as deleted?

```suggestion
	Logger(const Logger&) = delete;
```

Same for the copy assignment operator below.

Reply:
Ok, then let's do some more cleanup in the logger ...